chains/1.json

```diff
@@ -230,4 +230,11 @@
     "decimals": 18,
     "logoURI": "../images/veogv.svg"
   }
+  {
+    "address": "0x18084fbA666a33d37592fA2633fD49a74DD93a88",
+    "name": "tBTC v2",
+    "symbol": "tBTCv2",
```

Comment:
[Etherscan](https://etherscan.io/token/0x18084fbA666a33d37592fA2633fD49a74DD93a88#readContract#F11) shows the symbol as being tBTC:

<img width="194" alt="image" src="https://user-images.githubusercontent.com/94649004/200600322-1224c8e0-0c04-4f9e-925e-f78eb20d7480.png">